src/src/system.c

```diff
@@ -139,6 +141,9 @@ bool otSysPseudoResetWasRequested(void)
 
 void otSysProcessDrivers(otInstance *aInstance)
 {
+#if OPENTHREAD_UART_RTT_ENABLE
+    utilsUartRttProcess();
+#endif
```

Comment:
Could we move it inside nrf5TransportProcess? I believe motivation here is to match e.g. UART functionality, right?

Reply:
Yes that was the intention